### DISCUSSION TITLE
Retreiving Password (Docker)

### DISCUSSION THREAD

**Discussion by MinamiHasaki** [firefly-iii/firefly-iii#7652] [Firefly III] [answered]
Hi,

I'm running Firely III on Ubuntu using docker. I've followed the instructions given here: #4039 but I don't get a link to reset my password. Instead I get this error: _local.ERROR: Cowardly refuse to send a password reset message to user #1 because the reset button has been throttled._

What does that mean and how do I fix it to be able to get a password reset link to reset my password to access Firefly?

Thanks you.

**Comment by JC5** [comment, marked as answer]
Waiting a few minutes usually does it!

**Reply by MinamiHasaki** [reply]
After waiting, the error doesn't show up in the logs, but the logs still do not show a password reset link.

**Reply by JC5** [reply]
Then it depends on your email settings I guess.

**Reply by MinamiHasaki** [reply]
I'll take another look at it then before bothering you again, thank you.

p.s. Firefly III is awesome! Thank you for making such and amazing product!

Edit: I'll be marking as answered since waiting did fix the initial error.

**Reply by JC5** [reply]
Let me know. It should be "log", if it's "array" nothing will ever show up. But bad smtp settings will fail. You should have noticed it in the logs tho. Let me know!
